src/datafactory/azext_datafactory/generated/action.py

```diff
@@ -101,6 +109,9 @@ def get_action(self, values, option_string):  # pylint: disable=no-self-use
             v = properties[k]
             if kl == 'name':
                 d['name'] = v[0]
+            else:
+                raise CLIError('Unsupported Key {} is provided for parameter folder. All possible keys are: name'.
```

Comment:
`CLIError` is deprecated

Reply:
Generated code in extensions still uses `CLIError` to be compatible with older version of CLI.